servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1497,16 +1495,27 @@ public final Completable ambWith(final Completable other) {
     // Conversion Operators End
     //
 
+    /**
+     * Returns the {@link AsyncContextMap} to be used for a subscribe.
+     *
+     * @param provider The {@link AsyncContextProvider} which is the source of the map
+     * @return {@link AsyncContextMap} for this subscribe operation.
+     */
+    protected AsyncContextMap contextForSubscribe(AsyncContextProvider provider) {
```

Comment:
`AsyncContextProvider` is package private (not intended to be exposed), this method should be package private too.